src/masternodes/mn_checks.cpp

```diff
@@ -3920,7 +3920,7 @@ Res ValidateTransferDomain(const CTransaction &tx,
 
         CTxDestination dest;
 
-        // Soruce validation
```

Comment:
Ignore stuff like this. This is now a difference between feature/evm and master. We can sort this out after merging feature/evm.